R/mod_analysis.R

```diff
@@ -235,7 +235,8 @@ mod_rql_button_server(
           shinyjs::enable("download_csv_ui_1-download", asis = TRUE)
           shinyjs::enable("download_html_ui_1-report", asis = TRUE)
           glob$segments_df <- loc$segments_df %>%
-            dplyr::select(doc_name, doc_id, code_name, code_id, segment_text, user_name)
+            dplyr::select(doc_name, doc_id, segment_start, code_name, 
```

Comment:
should we not include segment_end while we are at it? possible use could be segment length calculation in spreadsheet softwares